package.json

```diff
@@ -1,44 +1,27 @@
 {
-  "name": "robert-website",
+  "name": "next-markdown-blog",
```

Comment:
Maybe we should update the project name to represent what it did before (robert-website)